subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/BuildScriptBuilder.java

```diff
@@ -1346,7 +1346,12 @@ public String firstArg(ExpressionValue argument) {
         @Override
         public String pluginDependencySpec(String pluginId, @Nullable String version) {
             if (version != null) {
-                return "id(\"" + pluginId + "\").version(\"" + version + "\")";
+                String kotlinPackagePrefix = "org.jetbrains.kotlin.";
+                if (pluginId.startsWith(kotlinPackagePrefix)) {
+                    return "kotlin(\"" + pluginId.substring(kotlinPackagePrefix.length()) + "\") version \"" + version + "\"";
+                } else {
+                    return "id(\"" + pluginId + "\") version \"" + version + "\"";
```

Comment:
I like this change, but let's leave the verboseness for kotlin plugins for now to keep it consistent with the Groovy syntax.

Reply:
Just a thought / asking out of curiosity: Why is it more important to keep similarity to Groovy syntax than applying "best practice" here for Kotlin syntax? I believe people are likely to use this code as a template, and we should care about spreading best practice rather than legacy syntax, or?

Reply:
It's one less thing that is different when applying plugins and one less thing we need to explain when someone is trying to understand where the `org.jetbrains.kotlin.jvm` plugin is coming from or which plugin `kotlin("jvm")` actually is.

There's some disagreement that the `kotlin()` shortcut is a good idea/best practice.

Reply:
Ok, I've dropped the commit.